test/RB_SQL/RB_Config.sql

```diff
@@ -0,0 +1,116 @@
+LOCK TABLES `Config` WRITE;
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (1,2,'1');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (2,3,'LORIS Demonstration Database');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (3,42,'<h3>Example Study Description</h3>\r\n <p>This is a sample description for this study, because it is a new LORIS install that has not yet customized this text.</p>\r\n <p>A LORIS administrator can customize this text in the configuration module, under the configuration option labeled \"Study Description\"</p>\r\n <h3>Useful Links</h3>\r\n <ul>\r\n <li><a href=\"https://github.com/aces/Loris\" >LORIS GitHub Repository</a></li>\r\n <li><a href=\"https://github.com/aces/Loris/wiki/Setup\" >LORIS Setup Guide</a></li>\r\n <li><a href=\"https://www.youtube.com/watch?v=2Syd_BUbl5A\" >A video of a loris on YouTube</a></li>\r\n </ul>');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (4,4,'images/LORIS_logo.png');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (5,5,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (6,6,'5');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (7,7,'99');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (9,9,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (10,10,'2016');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (11,11,'2028');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (12,13,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (13,14,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (15,16,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (16,20,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (17,21,'true');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (18,22,'true');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (19,23,'0');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (20,24,'Customize this text with your citation policy (via Configuration module)');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (22,12,'Ymd');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (23,27,'/data-raisinbread/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (24,28,'/var/www/loris/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (26,30,'/PATH/TO/EXTERNAL/LIBRARY/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (28,32,'/data/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (29,33,'tools/logs/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (30,34,'/data/incoming/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (31,35,'/data/demo/bin/mri/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (32,36,'/data/incoming/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (33,37,'/PATH/TO/Genomic-Data/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (34,38,'/data/uploads/');
```

Comment:
ConfigID 35 (MRICodePath), 27 (imagingPath) are essentials and not set. Would need to set them here.

Reply:
in the end, found the configs.